types/schema.d.ts

```diff
@@ -1,4 +1,4 @@
-import { ValidatorCompiler } from '@fastify/ajv-compiler'
+import { ValidatorFactory } from '@fastify/ajv-compiler'
```

Comment:
I was not that sure, regarding this typing change.

Reply:
But isn't it the same?

https://github.com/fastify/ajv-compiler/blob/c257dc65ed2b1e9213303fc8caec39ff40566a3b/types/index.d.ts#L54

Reply:
No. The reason why it currently doesnt work properly is that the ValidatorFactory instantiates  ValidatorCompilers.

The reason why i am unsure is that i would like to have atleast a typing test.  Other than that i think it should be the ValidatorFactory.